components/form/UploadImageInput.tsx

```diff
@@ -11,7 +11,9 @@ type UploadImageProps = {
 };
 
 const CDN =
-  'https://undfcbmldjkujposixvn.supabase.co/storage/v1/object/public/images/';
+  process.env.NODE_ENV === 'production'
+    ? 'https://undfcbmldjkujposixvn.supabase.co/storage/v1/object/public/images/'
+    : 'http://127.0.0.1:54321/storage/v1/object/public/images/';
```

Comment:
This is genius! But I actually needed to make a couple of changes on my end to make this work:

In my `Studio URL` I needed to create a bucket by going onto `storage > new bucket`
![Screenshot 2024-07-29 at 22 21 44](https://github.com/user-attachments/assets/60f639d8-4144-48a8-94c3-65ce7529299b)

Before I did that the images were not showing anywhere, but after this step I was able to upload them successfully ✨ 

Did you need to do the same in your local environment?

Reply:
That makes sense. We'll definitely need to record this somehow to make sure everyone has the same bucket once we merge these changes then.

Weirdly enough though I've just retried creating the bucket and uploading images and now they are showing up in the bucket but won't display on the page for some reason 🤔 they were showing just fine when I did my first review however.

Reply:
Would you try again please? Mine are working ok at the moment

Reply:
It was the `public` setting on my bucket that was giving trouble - sorry 😬